src/SPECS/hootenanny.spec.in

```diff
@@ -206,6 +212,15 @@ if [ "$1" = "1" ]; then
         sudo -u postgres psql -d wfsstoredb -c 'create extension postgis;'
     fi
 
+    # restore saved db config file settings if present
+    if [ -f /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave ]; then
+        if [ -f /var/lib/hootenanny/conf/DatabaseConfigDefault.sh ]; then
+            sudo grep DB_PASSWORD /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave > /var/lib/hootenanny/conf/DatabaseConfigLocal.sh
+        else
+            sudo mv /var/lib/hootenanny/conf/DatabaseConfig.sh.rpmsave /var/lib/hootenanny/conf/DatabaseConfig.sh
+        fi
+    fi
+
```

Comment:
This code is in the post yum install section, but handles the case where a hoot package was removed and then installed again.